test/xskbench/xskbench.c

```diff
@@ -1453,7 +1496,9 @@ ParseQueueArgs(
             if (++i >= argc) {
                 Usage();
             }
-            Queue->umemsize = atoi(argv[i]);
+            if (!ParseUInt64A(argv[i], &Queue->umemsize)) {
```

Comment:
There's not standard solution for this??

Reply:
Agreed, shocking. This is what Jeffrey suggested as his standard solution in 2020; I trust that he would have been aware of better standard solutions. IDK if things have improved since then.

Reply:
Copilot says:
```C
    char *endptr;
    uint64_t value;

    // Check if the string starts with "0x" or "0X" for hexadecimal
    if (str == '0' && (str == 'x' || str == 'X')) {
        value = strtoull(str, &endptr, 16);
    } else {
        value = strtoull(str, &endptr, 10);
    }
```

Reply:
Seems like it would work, though IDK if it is any better? It's definitely more lines of code.